administrator/components/com_content/src/Model/ArticleModel.php

```diff
@@ -477,17 +477,19 @@ public function getItem($pk = null)
 	 *
 	 * @param   array    $data      Data for the form.
 	 * @param   boolean  $loadData  True if the form is to load its own data (default case), false if not.
+	 * @param   boolean  $clear     Optional. Default false. True if don't want the cached form.
 	 *
 	 * @return  Form|boolean  A Form object on success, false on failure
 	 *
 	 * @since   1.6
 	 */
 	public function getForm($data = array(), $loadData = true)
 	{
+		$clear = (func_num_args() > 2) ? func_get_arg(2) : false;
```

Comment:
@rs4231199 Why not add the `$clear` as third optional argument in the `getForm()` function?